### ISSUE TITLE
A problem about training and testing on Dunhuang dataset

### ISSUE THREAD

**Issue by mmmjmmm** [ChrisChen1023/HINT#24]
I've download the dunuhuang pre-trained model from your drive, and I saw your iteration is about 300k. But when I tried to load the checkpoints and continue training on your work, I found that the iteration start from 24000, is that correct?
![image](https://github.com/user-attachments/assets/82850a5a-afaa-4e41-889d-08adcd6b352b)
And what should I set for the 'MASK' config in config.yml if I try to train on dunhuang dataset?
![image](https://github.com/user-attachments/assets/b7781527-d915-4d9c-b174-78acbec291d1)
By the way, why my testing result using your pre-trained model on dunhuang dataset is not good as you show in the paper? 
![image](https://github.com/user-attachments/assets/a9bbb84d-9e77-4085-8d97-c214ddd378b5)
![image](https://github.com/user-attachments/assets/b5cd5c8e-52bd-4dab-9bc9-bd399b186439)
Thank you for your work.
Looking forward to your reply.

**Comment by Sheeran2000** [comment]
I  want  ask  how to train the HINT？

**Comment by ChrisChen1023** [comment]
> I've download the dunuhuang pre-trained model from your drive, and I saw your iteration is about 300k. But when I tried to load the checkpoints and continue training on your work, I found that the iteration start from 24000, is that correct? ![image](https://private-user-images.githubusercontent.com/80912258/380627998-82850a5a-afaa-4e41-889d-08adcd6b352b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4NTI4NTYsIm5iZiI6MTczNjg1MjU1NiwicGF0aCI6Ii84MDkxMjI1OC8zODA2Mjc5OTgtODI4NTBhNWEtYWZhYS00ZTQxLTg4OWQtMDhhZGNkNmIzNTJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTE0VDExMDIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4MWQ4MTc0NDJiMDZlZWI1N2M2MDAzNDAyMDI0NGFjNWEyNjA3ZGRlNGY4NzAxYWVjNTAyZjJhN2U3MmNkNWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.eioaosK_pt9VtJ3ALPbwS4WImZAb4RmPR6Z4kRSuz44) And what should I set for the 'MASK' config in config.yml if I try to train on dunhuang dataset? ![image](https://private-user-images.githubusercontent.com/80912258/380628516-b7781527-d915-4d9c-b174-78acbec291d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4NTI4NTYsIm5iZiI6MTczNjg1MjU1NiwicGF0aCI6Ii84MDkxMjI1OC8zODA2Mjg1MTYtYjc3ODE1MjctZDkxNS00ZDljLWIxNzQtNzhhY2JlYzI5MWQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTE0VDExMDIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxM2U4NzhmMGU2NWQ0YWFmNTczYmRjMmY2MThkNDRjNWYxNjExOGU2M2I5YmFjNzU5ZGExNTFkN2M1YzFjMjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UXERr1rGhaoJKGSJyABFqHkeuqd46ZXAGSUbbeIWt8I) By the way, why my testing result using your pre-trained model on dunhuang dataset is not good as you show in the paper? ![image](https://private-user-images.githubusercontent.com/80912258/380629356-a9bbb84d-9e77-4085-8d97-c214ddd378b5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4NTI4NTYsIm5iZiI6MTczNjg1MjU1NiwicGF0aCI6Ii84MDkxMjI1OC8zODA2MjkzNTYtYTliYmI4NGQtOWU3Ny00MDg1LThkOTctYzIxNGRkZDM3OGI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTE0VDExMDIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiNTMyMTQzOTZmMjQzMjZkYjg5ODMzMzRmNjRhNWZhNzNhMzJkNmVhMGI4NTFkM2UzNTU2NWRhNWEyNTBkMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.oNlt-EaGG3cW3PVmSWx6vr-NEaj0Dfl-8wtI7dYwxw8) ![image](https://private-user-images.githubusercontent.com/80912258/380629509-b5cd5c8e-52bd-4dab-9bc9-bd399b186439.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4NTI4NTYsIm5iZiI6MTczNjg1MjU1NiwicGF0aCI6Ii84MDkxMjI1OC8zODA2Mjk1MDktYjVjZDVjOGUtNTJiZC00ZGFiLTliYzktYmQzOTliMTg2NDM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTE0VDExMDIzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlMmEwMjM3MzVmYzYxMzVhNjZiY2U3MTUwZDU0MjhjODQ3ZDNkNTkyNzJlOTQ0ZjFhMDVjMDhkNDIxZTI2NzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.E7ZPBxdxtmHYw2W245zvArHQ8SlKramNUHVU6JXOynY) Thank you for your work. Looking forward to your reply.

Hi,

Thanks for your interests. I double-checked our record, the result on the paper aligns with our record.
![image](https://github.com/user-attachments/assets/2fdee903-b029-4948-8708-66ccab5bd758)
The reason why you did not get the correct result might be, we upload the wrong version for pre-trained model, which we truly apologise. That's said, unfortunately, I get a new academic role now and unable to find the previous pre-trained models. therefore I retrain the model, the results is much close to what we present on the paper now. 
Sorry for the inconvenience and late reply.
